### PR TITLE
feat(ui): Rename resolved in options, add analytics

### PR DESCRIPTION
**Pull request by matejminar**
- Renaming option from `Another version...` to `Another existing release`.
- Add usage tracking

image

### CODE REVIEW

File: static/app/components/actions/resolve.tsx [hunk 1]
```diff
@@ -165,12 +193,12 @@ class ResolveActions extends React.Component<Props> {
 
         <MenuItemActionLink
           {...actionLinkProps}
-          title={t('Another version')}
+          title={t('Another existing release')}
```
Comment (billyvg) [inline static/app/components/actions/resolve.tsx:196]: is `existing` needed? Seems a bit redundant. What is a non-existent release?

File: static/app/components/actions/resolve.tsx [hunk 2]
```diff
@@ -184,7 +212,7 @@ class ResolveActions extends React.Component<Props> {
       <CustomResolutionModal
         {...deps}
         onSelected={(statusDetails: ResolutionStatusDetails) =>
-          this.onCustomResolution(statusDetails)
+          this.handleAnotherExistingReleaseResolution(statusDetails)
```
Comment (billyvg) [inline static/app/components/actions/resolve.tsx:215]: Thanks for changing these :)

**Review by billyvg — APPROVED** [review]
LGTM aside from `existing`

File: static/app/components/actions/resolve.tsx (hunk 1 above)
Comment (matejminar) [reply static/app/components/actions/resolve.tsx:196]: A couple of customers and even some employees thought that they can enter there any version name (even future one) in which the issue will be resolved. (Especially in semver world)

**Pull request merged**